lua/entities/gmod_wire_expression2/core/custom/cfc_entity.lua

```diff
@@ -52,4 +52,21 @@ e2function void entity:ejectPodTo(vector pos)
 
     driver:ExitVehicle()
     driver:SetPos(clampedPos)
-end
\ No newline at end of file
+end
+
+
+-- Easy way to have some of the setPos benefits without mostly being able to be abused.
+e2function number entity:setFairPos(vector pos)
+    if this:IsPlayer() then return 3 end -- Debug output 3 if the target is a player, because we dont want someone tping themselves around the e2 or any other player.
+    local posToLoc = pos - self:GetPos() -- distance to the wanted position
+    local lengthPos = posToLoc[1]^2 + posToLoc[2]^2 + posToLoc[3]^2 -- we dont need to do the square root on this, itll just waste time and resources
```

Comment:
Of all of the things you leave comments on, you neglected to explain what this math actually _does_. The name of the variable should explain what the result of the math is, and the comment should explain the methodology behind getting that number.